qa/tomcat-runtime/pom.xml

```diff
@@ -145,5 +138,17 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>engine-integration-jakarta</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.jboss.weld.servlet</groupId>
+          <artifactId>weld-servlet-shaded</artifactId>
```

Comment:
`Weld` is not needed when the `webapp-integration-tests` are executed. As a result, adding the dependency on `tomcat-runtime` breaks the tests.

This line moves the addition of the dependency only when `engine-integration-jakarta` is invoked